### PR TITLE
Create a service class to access localized messages.

### PR DESCRIPTION
**Pull request by DenysBurduzhan**

### CODE REVIEW

**Comment by DenysBurduzhan** [comment]
Если правильно понял, в тикете создаем пока просто класс. Заполняем файлы текстом потом?

**Review by dRaider-bitrek — APPROVED** [review]

**Comment by dRaider-bitrek** [comment]
@oleksandr-jr Зроби будласка review.

File: src/main/java/ua/com/javarush/gnew/m2/service/LocalizationService.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+
+    public String getMessage(String key) {
+        try {
+            return resourceBundle.getString(key);
+        } catch (Exception e) {
+            return "Message not found for key: " + key;
+        }
+    }
```
Comment (oleksandr-jr) [inline src/main/java/ua/com/javarush/gnew/m2/service/LocalizationService.java:26]: Maybe it's a good idea to add a `fallback` locale here in the case we can't find this message in the current locale.

Example:
We are looking for a Ukrainian word 'Котактʼ using key, but we can't find it. Then we simply show this word in English before throwing an exception. So in this way we reduce the amount of exceptions and have a “base language”.

**Review by oleksandr-jr — CHANGES_REQUESTED** [review]
In general, looks good, but there are a couple of things.
1. Fallback language (Highly suggested)
2. Accessibility of the `getMessage` method.
3. I'm not sure about multithreading here. Maybe you will need to deal with changes later.

Please think on how you will use this class, more specifically, how you get an object of this class all around the project. Currently, you have to create one every time or inject it.

Approaches to consider:
1. Global context class (Maybe already implemented)
2. Singleton
4. Static utility class
5. Implement some kind of Dependency Injection.
6. Service Locator Pattern

**Review by oleksandr-jr — APPROVED** [review]

**Pull request merged**